src/TargetDescription.cc

```diff
@@ -0,0 +1,128 @@
+#include "TargetDescription.h"
+#include "GdbServerConnection.h"
+#include "kernel_abi.h"
+#include <sstream>
+
+namespace rr {
+
+class FeatureStream {
+public:
+  std::string result() { return stream.str(); }
```

Comment:
Add `using namespace std;` after the includes and don't use the `std::` prefix (except for `std::move` where clang requires it...)